src/resources/js/views/rsvp.js

```diff
@@ -44,38 +38,39 @@
 	 *
 	 * @return void
 	 */
-	$( '.tribe-block__rsvp__ticket' ).on( 'click',
-		'.tribe-block__rsvp__status-button--going, .tribe-block__rsvp__status-button--not-going',
-		function( e ) {
-
-			var button    = $( this );
-			var ticket    = $( this ).closest( '.tribe-block__rsvp__ticket' );
-			var ticket_id = ticket.data( 'rsvp-id' );
-			var going     = button.hasClass( 'tribe-block__rsvp__status-button--going' );
+	function handle_rsvp_response() {
```

Comment:
Same for all the other methods, would love to store em on an object. It does make our life a ton easier later on for customization from other plugins or extensions.